README.md

```diff
@@ -198,6 +198,15 @@ Press <kbd>F10</kbd> to step to the next statement, or <kbd>F5</kbd>
 again to continue until the next breakpoint, or termination of the
 program.
 
+## Coverage
+
+To enable coverage you can set `enable-coverage` to `true` in your
```

Comment:
Could you reformulate into two distinct paragraphs that focus on one element each?
The first one focuses on coverage files, the second one on highlighting in the interface.

In addition, for now the `README.md` does not mention `task.json` or `launch.json` (I think), as most features described should work by default without those files (this is for ease of access to VS Code beginners). If we don't enable coverage by default (maybe that's a good idea to have to enable it), then we need paragraphs stating that those files need to be constructed via (`Run > Configure build tasks…` or similar commands).

Reply:
https://github.com/OCamlPro/superbol-studio-oss/blob/e2bd9a73ec3c94dd4792d3867e441e63077ffc68/README.md?plain=1#L173
The `README.md` does mention the `task.json` file, I'll edit the paragraph to focus on both features in distinct way tho

Reply:
`Setting up a Build Task for Debugging` is out of date: by default we don't need the `task.json` anymore.

Reply:
You can probably reuse part of that paragraph down in `Coverage`.

Reply:
So I should delete `Setting up a Build Task` paragraph? I think maybe, if the README is outdated, a full PR to update it should be the way to go.
It would be better than modifying parts of it when we notice the deprecation